webapp/commonComponents/validationTooltip.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+import * as ValidationUtils from '../utils/validationUtils'
+
+import * as R from 'ramda'
+
+import Tooltip from './tooltip'
+
+import useI18n from '../commonComponents/useI18n'
+
+export default ({ validation, children, className }) => {
+  const i18n = useI18n()
+
+  const messagesHtml = ValidationUtils.getValidationFieldMessagesHTML(i18n)(validation)
+  const type = R.isEmpty(validation) ? '' : 'error'
```

Comment:
should this condition include ```Validator.isValidationValid```  as well?
i.e. ```R.isEmpty(validation) || Validator.isValidationValid(validation) ? '' : 'error' ```

Reply:
I think that the condition R.isEmpty(validation) || Validator.isValidationValid(validation) should be evaluated before generating the messageHtml, because in that case there won't be any messages to show.

Reply:
removed R.isEmpty(validation) and passing validation from errorBadge; Checking validity before generating messagesHtml